pootle/core/views.py

```diff
@@ -464,7 +465,8 @@ def is_admin(self):
 
     @property
     def language(self):
-        return None
+        if self.tp:
+            return self.tp.language
```

Comment:
because it implements the basic functionality - which most if not all child classes share i believe.